### ISSUE TITLE
Closing menu while passing over an other menu - Bis

### ISSUE THREAD

**Issue by yvideau** [joeldbirch/superfish#77]
SORRY I closed the issue by mistake.

Hi Joeldbirch,

I've basically build a vertical menu where the submenu open below each menu as seen on the screen shoot below.

![superfish issue - printscreen 1](https://f.cloud.github.com/assets/4543942/573233/9a48547a-c7ac-11e2-9945-6c0e0cc149d6.jpg)

I used SF v1.7.2 / HI r7 / jQuery v1.9.1
I used show/hide height as animation with 1000 ms speed/speedOut

Now lets say that the level1 menu is open, and we move the mouse to the level 2 menu
--> the level 2 menu start to open and the level 1 menu start to close.
Because the level 1 submenus collapse the level 2 and level 3 menu are changing position i.e moving up.

![superfish issue - printscreen 2](https://f.cloud.github.com/assets/4543942/573241/aeb312ce-c7ac-11e2-8766-cb16ad7a582f.jpg)

If the user is not moving the mouse, the pointer will then pass hover the level 3 menu
--> the level 1 "fully collapse in no time" (not natural behavior)
the level 2 menu stop opening and start closing (natural behavior)
the level 3 menu start opening (natural behavior).

I was using previously superfish v1.4.8 and the hoverIntent which was release at that time, plus jQuery v1.7.2
--> the problem was not happening i.e level 1 and 2 menus "were closing in the set time" while level 3 menu was opening.

In other words it seem that with this release of SF and/or HI opening and closing multiple menus at the same time is not possible.
The problem is that the current behavior is not userfriendly, the fact the level 1 menu collapse instantaneously is very disruptive.

Any idea was cause the issue in SF and/or HI.
Thanks.
